src/material.js

```diff
@@ -144,6 +150,10 @@ export class Material {
         return this.prop('basis', undefined, true);
     }
 
+    set basis(config) {
```

Comment:
There's `setBasis` already above

Reply:
There is but it takes in 3 parameters. I want a function that will take in a basis that already exists and apply it a material.

The other function seems more appropriate if you're trying to update based on content from a text file or from changes in the basis editor.

Reply:
OK